app/forms/new_article_form.rb

```diff
@@ -0,0 +1,12 @@
+class NewArticlesForm
+    include ActiveModel::Model
+
+
+    attr_accessor(
+        :title,
+        :text
+    )
+
+    validates :title, presence: true
+    validates :text, presence: true
+end
```

Comment:
check this red sign out:
![image](https://user-images.githubusercontent.com/11631898/78819665-3f9ad100-79df-11ea-83b7-d3d7cc08d50e.png)

it means that your editor doesn't set up End Of File character correctly,
try searching about how to set it up for your editor specifically